source/_posts/2018-11-13-whining-vs-venting.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Whining vs Venting"
+date: 2018-11-13 08:14:11 -0700
+comments: false
+categories: management
+---
+It was time for my one-on-one with Ron and I could tell it would be a doozy. He slumped into a chair and before I could even start with a basic “how’s it going?” he launched into a tirade about Sales and how they were screwing us over *again* and that he had to pick up the extra load. This is a not exactly a unique case - there’s a natural tension between promise and delivery and there’s more than a kernel of truth to it. Ron was really steamed this time though. It could be he needed to vent, but maybe he was just whining.
```

Comment:
Replaced `-` with `&mdash;`; remove spaces before and after em dash.
Comma between `promise` and `and`.
Comma between `time` and `though` (I think--this might be optional, can't entirely remember the rule).